docker-for-mac/release-notes.md

```diff
@@ -16,11 +16,44 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop Community 3.2.0
+2021-03-NN
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### New
+
+- The dashboard shows a tip of the week.
+- The dashboard automatically opens when starting Docker Desktop.
+- Add experimental support to use the new Apple Virtualization framework (requires macOS Big Sur 11.1 or later)
+- Use iTerm2 to do `docker exec` if iTerm2 is installed, if not then use the default Terminal.App [docker/for-mac#4273](https://github.com/docker/for-mac/issues/4273)
+
+### Upgrades
+
+- [Docker Engine 20.10.4](https://docs.docker.com/engine/release-notes/#20104)
+- [Docker Compose 1.28.4](https://github.com/docker/compose/releases/tag/1.28.4)
+- [Compose CLI v1.0.8](https://github.com/docker/compose-cli/tree/v1.0.8)
+- [Docker Hub Tool v0.3.0](https://github.com/docker/hub-tool/releases/tag/v0.3.0)
+- [QEMU 5.0.1](https://wiki.qemu.org/ChangeLog/5.0)
```

Comment:
This was a downgrade not an upgrade, wasn't it? Or did we never ship 5.2?

Reply:
Yes, we shipped 5.2 only in the M1 preview, but not in the official releases yet.